{{ cookiecutter.dir_name }}/build.gradle

```diff
@@ -0,0 +1,40 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.2.0'
+    }
+}
+
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 25
+    buildToolsVersion "25.0.2"
```

Comment:
Hrm. You may well be correct - the installation process for the android tools is such a cluster :duck: - I possibly lost track of what was and what wasn't being installed for me. I'll need to try a clean-room install at some point to make sure we have the instructions right.